lib/segment/src/fixtures/payload_context_fixture.rs

```diff
@@ -119,6 +119,14 @@ impl IdTracker for FixtureIdTracker {
         )
     }
 
+    /// Not actually random, but it's enough for tests
+    fn iter_random(&self) -> Box<dyn Iterator<Item = (PointIdType, PointOffsetType)> + '_> {
+        Box::new(
+            self.iter_ids()
+                .map(|id| (PointIdType::NumId(id as u64), id)),
+        )
+    }
```

Comment:
Why not proper random?

Reply:
tbh, laziness, but I can just copy the implementation from the id-tracker :)

Reply:
left it at unimplemented